README.md

```diff
@@ -1,2 +1,11 @@
 # virtio-bindings
-Virtual IO (virtio) kernel bindings
+Rust FFI bindings to virtio generated using [bindgen](https://crates.io/crates/bindgen).
+
+# Usage
+Add this to your `Cargo.toml`:
+```toml
+virtio-bindings = { version = "0.1", features = ["virtio-v5_0_0"]}
+```
```

Comment:
Please also add a one liner to describe how a crate should include these bindings:

```Rust
extern crate virtio_bindings;
```